avocado/core/varianter.py

```diff
@@ -416,11 +416,11 @@ def is_parsed(self):
         """
         return self.variants is not None
 
-    def _skip_new_data_check(self, fction, args):
+    def _skip_new_data_check(self, function, args):
```

Comment:
It's either that or add "fction" to the spelling ignore list.  So yes, I prefer it like `fction`.